_docs/style/glossary.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Glossary
+layout: docs
+category: Style
+usage: |
+  The following is a glossary of common terms found in the Helix documentation.
```

Comment:
s/`The following is a glossary of common terms found in the Helix documentation.`/`This glossary provides common terms found in the Helix documentation.` (see the last row in http://rackerlabs.github.io/docs-rackspace/style-guide/style/lists.html#introductory-text)